### PR TITLE
Swift 4.2 Compatibility Fixes

### PR DESCRIPTION
**Pull request by jpmendel**
- Updated Podfile to reflect Rx spec dependencies.

### CODE REVIEW

**Comment by jpmendel** [comment]
Since we haven't pushed this with trunk yet, I'm going to try and re-tag 0.11.0 onto these commits.

File: Podfile [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+platform :ios, '9.0'
 
 use_frameworks!
 inhibit_all_warnings!
 
 def commonpods
   pod 'IP-UIKit-Wisdom'
-    pod 'RxSwift', '~> 4.0'
-    pod 'RxCocoa', '~> 4.0'
+    pod 'RxSwift', '~> 4.3.0'
```
Comment (bobgilmore) [inline Podfile]: Do you want this (and all of the others) to be `~> 4.3.0`, or `~> 4.3`?

Comment (paulrolfe) [reply Podfile]: For posterity capturing an offline convo: Do the same change for the podspec.

File: Podfile.lock [hunk 2]
```diff
@@ -25,6 +25,6 @@ SPEC CHECKSUMS:
   RxSwift: fe0fd770a43acdb7d0a53da411c9b892e69bb6e4
   RxTest: ea97a208826906f3904c0debdd09575ebcbfe216
 
-PODFILE CHECKSUM: d81cbe2f98140887848195204b8b667819be7198
+PODFILE CHECKSUM: c9e34e17eadb5b684ed6407d1d2153a14382d11b
 
-COCOAPODS: 1.5.3
+COCOAPODS: 1.6.0.beta.1
```
Comment (paulrolfe) [inline Podfile.lock]: Can this be downgraded to a non-beta?

**Review by paulrolfe — APPROVED** [review]

**Review by Kevin-Monahan-Intrepid — APPROVED** [review]

**Review by bobgilmore — APPROVED** [review]

**Pull request merged**